examples/asr/librispeech_ctc_decoder/inference.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import logging
+from typing import Optional
+
+import torch
+import torchaudio
+from torchaudio.prototype.ctc_decoder import kenlm_lexicon_decoder
+
+
+logging.basicConfig(format="%(asctime)s %(message)s", level=logging.INFO, datefmt="%Y-%m-%d %H:%M:%S")
```

Comment:
Since this is a standalone script, this is fine, but in general, the initialization of logger module should be delayed until the control flow reaches the entrypoint. That way, logger can be customized based on the CLI options.

```python
def _main():
    args = _parse_args()
    _init_logger(args.debug)
    run_inference(args)

def _parse_args():
    parser = ArgumentParser(...)
    parser.add_argument('--debug', action='store_true')
    ...
    return parser.parse_args()

def _init_logger(debug):
    fmt = '<more metadata>' if debug else 'simple'
    level = logging.DEBUG if debug else logging.INFO
    logging.basicConfig(format=fmt, level=level)

if __name__ == '__main__':
    _main()
```